### DISCUSSION TITLE
babylon 13  not worked with me , how can i fix this issue whith sass ,css  the project cant read it

### DISCUSSION THREAD

**Discussion by Yazan-Abuawwad** [primefaces/community#603] [PrimeFaces Templates] [answered]
my project spring boot 3 +java 17+ jsf 2.3 + primeface 13+ babylon 13  
![Capture](https://github.com/primefaces/community/assets/88251354/4289645e-6ac5-4dc3-b9a2-e30b276e7de8)
css not work 
sass installed by doc  but does not work 
![Capture2](https://github.com/primefaces/community/assets/88251354/575c98e5-a97d-415b-bdab-f828f99ee60a)
all XML files like web in WEB-INF as it is, I have not modified it
my pom.xml like this  if I need to add dep or remove feel free to tell me
`<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>3.2.0</version>
        <relativePath/> 
    </parent>

    <groupId>org.primefaces</groupId>
    <artifactId>babylon</artifactId>
    <version>7.0.0</version>
     <packaging>${packaging.type}</packaging>

    <name>babylon</name>

    <properties>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <owb.version>2.0.20</owb.version>
        <mojarra.version>2.3.14</mojarra.version>
        <open-pdf.version>1.3.30</open-pdf.version>
        <poi.version>5.0.0</poi.version>
        <packaging.type>war</packaging.type>
    </properties>

    <dependencies>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-freemarker</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-jersey</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-mustache</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-thymeleaf</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web-services</artifactId>
        </dependency>
        <dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-webflux</artifactId>
        </dependency>

        <dependency>
            <groupId>org.primefaces</groupId>
            <artifactId>primefaces</artifactId>
            <version>13.0.0</version>
             </dependency>
        <dependency>
            <groupId>org.glassfish</groupId>
            <artifactId>jakarta.faces</artifactId>
            <version>${mojarra.version}</version>
        </dependency>
        <dependency>
            <groupId>javax.el</groupId>
            <artifactId>javax.el-api</artifactId>
            <version>3.0.0</version>
            <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>javax.servlet</groupId>
            <artifactId>javax.servlet-api</artifactId>
            <version>4.0.1</version>
            <scope>provided</scope>
        </dependency>

        
        <dependency>
            <groupId>org.apache.geronimo.specs</groupId>
            <artifactId>geronimo-atinject_1.0_spec</artifactId>
            <version>1.0</version>
        </dependency>
        <dependency>
            <groupId>org.apache.geronimo.specs</groupId>
            <artifactId>geronimo-jcdi_2.0_spec</artifactId>
            <version>1.1</version>
        </dependency>
        <dependency>
            <groupId>org.apache.geronimo.specs</groupId>
            <artifactId>geronimo-interceptor_1.2_spec</artifactId>
            <version>1.0</version>
        </dependency>
        <dependency>
            <groupId>org.apache.geronimo.specs</groupId>
            <artifactId>geronimo-annotation_1.3_spec</artifactId>
            <version>1.0</version>
        </dependency>

        
        <dependency>
            <groupId>org.apache.openwebbeans</groupId>
            <artifactId>openwebbeans-impl</artifactId>
            <version>${owb.version}</version>
        </dependency>
        <dependency>
            <groupId>org.apache.openwebbeans</groupId>
            <artifactId>openwebbeans-web</artifactId>
            <version>${owb.version}</version>
        </dependency>
        <dependency>
            <groupId>org.apache.openwebbeans</groupId>
            <artifactId>openwebbeans-jsf</artifactId>
            <version>${owb.version}</version>
        </dependency>

        
        <dependency>
            <groupId>org.apache.poi</groupId>
            <artifactId>poi</artifactId>
            <version>${poi.version}</version>
        </dependency>
        <dependency>
            <groupId>org.apache.poi</groupId>
            <artifactId>poi-ooxml</artifactId>
            <version>${poi.version}</version>
            <exclusions>
                <exclusion>
                    <groupId>org.apache.xmlgraphics</groupId>
                    <artifactId>batik-all</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>de.rototor.pdfbox</groupId>
                    <artifactId>graphics2d</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.apache.santuario</groupId>
                    <artifactId>xmlsec</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.bouncycastle</groupId>
                    <artifactId>bcpkix-jdk15on</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>com.github.virtuald</groupId>
                    <artifactId>curvesapi</artifactId>
                </exclusion>
                <exclusion>
                    <groupId>org.bouncycastle</groupId>
                    <artifactId>bcprov-jdk15on</artifactId>
                </exclusion>
            </exclusions>
        </dependency>

        <dependency>
            <groupId>com.github.librepdf</groupId>
            <artifactId>openpdf</artifactId>
            <version>${open-pdf.version}</version>
        </dependency>
    </dependencies>

    <repositories>
        <repository>
            <id>prime-repo</id>
            <name>PrimeFaces Maven Repository</name>
            <url>http://repository.primefaces.org</url>
        </repository>
    </repositories>

    <build>
        <resources>
            <resource>
                <directory>src/main/resources</directory>
                <filtering>true</filtering>
            </resource>
            <resource>
                <directory>${basedir}/src/main/webapp/WEB-INF</directory>
                <includes>
                    <include>web.xml</include>
                </includes>
                <filtering>true</filtering>
                <targetPath>${project.build.directory}</targetPath>
            </resource>
        </resources>
        <plugins>

            <plugin>
                <groupId>org.springframework.boot</groupId>
                <artifactId>spring-boot-maven-plugin</artifactId>
                <version>3.2.0</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>repackage</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>

            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.7.0</version>
                <configuration>
                    <source>1.8</source>
                    <target>1.8</target>
                    <encoding>${project.build.sourceEncoding}</encoding>
                </configuration>
            </plugin>
            <plugin>
                <groupId>org.eclipse.jetty</groupId>
                <artifactId>jetty-maven-plugin</artifactId>
                <version>9.4.36.v20210114</version>
                <configuration>
                    <webAppConfig>
                        <contextPath>/babylon</contextPath>
                    </webAppConfig>
                    <scanIntervalSeconds>5</scanIntervalSeconds>
                    <webXml>${project.build.directory}/web.xml</webXml>
                </configuration>
            </plugin>

        </plugins>
    </build>

    <profiles>
        <profile>
            <id>distribute</id>
            <properties>
                <packaging.type>jar</packaging.type>
            </properties>
            <build>
                <plugins>

                    <plugin>
                        <artifactId>maven-assembly-plugin</artifactId>
                        <version>2.3</version>
                        <executions>
                            <execution>
                                <id>build-layout</id>
                                <phase>package</phase>
                                <goals>
                                    <goal>single</goal>
                                </goals>
                                <configuration>
                                    <descriptors>
                                        <descriptor>src/assemble/layout.xml</descriptor>
                                    </descriptors>
                                    <finalName>${project.name}-layout-${project.version}</finalName>
                                    <appendAssemblyId>false</appendAssemblyId>
                                </configuration>
                            </execution>
                            <execution>
                                <id>build-theme</id>
                                <phase>package</phase>
                                <goals>
                                    <goal>single</goal>
                                </goals>
                                <configuration>
                                    <descriptors>
                                        <descriptor>src/assemble/theme.xml</descriptor>
                                    </descriptors>
                                    <finalName>${project.name}-theme-${project.version}</finalName>
                                    <appendAssemblyId>false</appendAssemblyId>
                                </configuration>
                            </execution>
                        </executions>
                    </plugin>

                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-shade-plugin</artifactId>
                        <configuration>
                            <finalName>${project.name}-theme-${project.version}-jakarta</finalName>
                            <shadedArtifactAttached>false</shadedArtifactAttached>
                            <createDependencyReducedPom>false</createDependencyReducedPom>
                            <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
                            <minimizeJar>true</minimizeJar>
                            <shadeSourcesContent>true</shadeSourcesContent>
                            <filters>
                                <filter>
                                    <artifact>*:*</artifact>
                                    <includes>
                                        <include>org/primefaces/${project.name}/component/*</include>
                                        <include>META-INF/MANIFEST.MF</include>
                                        <include>META-INF/faces-config.xml</include>
                                        <include>META-INF/primefaces-${project.name}.taglib.xml</include>
                                        <include>META-INF/resources/primefaces-${project.name}-*/*</include>
                                    </includes>
                                </filter>
                            </filters>
                        </configuration>
                        <executions>
                            <execution>
                                <id>jakarta</id>
                                <phase>package</phase>
                                <goals>
                                    <goal>shade</goal>
                                </goals>
                                <configuration>
                                    <shadedArtifactAttached>true</shadedArtifactAttached>
                                    <shadedClassifierName>jakarta</shadedClassifierName>
                                    <createDependencyReducedPom>false</createDependencyReducedPom>
                                    <transformers>
                                        <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
                                        <transformer implementation="io.yupiik.maven.shade.transformer.RelocationTransformer">
                                            <delegates>
                                                <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer"/>
                                                <transformer implementation="org.apache.maven.plugins.shade.resource.XmlAppendingTransformer">
                                                    <resource>META-INF/faces-config.xml</resource>
                                                </transformer>
                                                <transformer implementation="org.apache.maven.plugins.shade.resource.XmlAppendingTransformer">
                                                    <resource>META-INF/primefaces-${project.name}.taglib.xml</resource>
                                                </transformer>
                                            </delegates>
                                        </transformer>
                                    </transformers>
                                    <artifactSet>
                                        <includes>
                                            <include>${project.groupId}:${project.artifactId}</include>
                                        </includes>
                                    </artifactSet>
                                    <relocations>
                                        <relocation>
                                            <pattern>javax.faces</pattern>
                                            <shadedPattern>jakarta.faces</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.el</pattern>
                                            <shadedPattern>jakarta.el</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.annotation</pattern>
                                            <shadedPattern>jakarta.annotation</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.servlet</pattern>
                                            <shadedPattern>jakarta.servlet</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.persistence</pattern>
                                            <shadedPattern>jakarta.persistence</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.enterprise</pattern>
                                            <shadedPattern>jakarta.enterprise</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.xml.bind</pattern>
                                            <shadedPattern>jakarta.xml.bind</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.validation</pattern>
                                            <shadedPattern>jakarta.validation</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.inject</pattern>
                                            <shadedPattern>jakarta.inject</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.activation</pattern>
                                            <shadedPattern>jakarta.activation</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.ws.rs</pattern>
                                            <shadedPattern>jakarta.ws.rs</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.jws</pattern>
                                            <shadedPattern>jakarta.jws</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.cache</pattern>
                                            <shadedPattern>jakarta.cache</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.xml.ws</pattern>
                                            <shadedPattern>jakarta.xml.ws</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>javax.xml.soap</pattern>
                                            <shadedPattern>jakarta.xml.soap</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>window.jsf</pattern>
                                            <shadedPattern>window.faces</shadedPattern>
                                        </relocation>
                                        <relocation>
                                            <pattern>jsf.ajax</pattern>
                                            <shadedPattern>faces.ajax</shadedPattern>
                                        </relocation>
                                    </relocations>
                                </configuration>
                            </execution>
                        </executions>
                        <dependencies>
                            <dependency>
                                <groupId>io.yupiik.maven</groupId>
                                <artifactId>maven-shade-transformers</artifactId>
                                <version>0.0.1</version>
                            </dependency>
                        </dependencies>
                    </plugin>

                </plugins>
            </build>
        </profile>
        <profile>
            <id>deploy</id>
            <activation>
                <activeByDefault>true</activeByDefault>
            </activation>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-war-plugin</artifactId>
                        <version>3.2.0</version>
                        <configuration>
                            <webResources>
                                <resource>
                                    <filtering>true</filtering>
                                    <directory>src/main/webapp</directory>
                                    <includes>
                                        <include>**/web.xml</include>
                                    </includes>
                                </resource>
                            </webResources>
                            <warSourceDirectory>src/main/webapp</warSourceDirectory>
                            <webXml>${project.build.directory}/web.xml</webXml>
                        </configuration>
                    </plugin>
                </plugins>
            </build>
        </profile>
    </profiles>

</project>
`

**Comment by mertsincan** [comment]
Interesting! Do you get an error on browser's console?  Did you try our sample project? Does it work for you?

**Reply by Yazan-Abuawwad** [reply]
![Capture3](https://github.com/primefaces/community/assets/88251354/0784ddf9-467b-429a-87ec-601e7681312b)
![Capture4](https://github.com/primefaces/community/assets/88251354/4e57f41a-4fbe-4d5d-a0ff-9e052458492b)
![Capture5](https://github.com/primefaces/community/assets/88251354/14c09695-80da-4620-82ff-0eeca1413e85)

**Reply by Yazan-Abuawwad** [reply]
and this i get with jetty:run 
`C:\Users\PC\.jdks\temurin-17.0.8.1\bin\java.exe "-Dmaven.multiModuleProjectDirectory=E:\work home\mo jaja\old\tag" "-Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\plugins\maven\lib\maven3" "-Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\plugins\maven\lib\maven3\bin\m2.conf" "-Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\plugins\maven\lib\maven-event-listener.jar" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\lib\idea_rt.jar=61527:C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\bin" -Dfile.encoding=UTF-8 -classpath "C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\plugins\maven\lib\maven3\boot\plexus-classworlds-2.6.0.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2022.2\plugins\maven\lib\maven3\boot\plexus-classworlds.license" org.codehaus.classworlds.Launcher -Didea.version=2022.2 org.eclipse.jetty:jetty-maven-plugin:9.4.36.v20210114:run
[INFO] Scanning for projects...
[INFO] 
[INFO] -----------------------< org.primefaces:babylon >-----------------------
[INFO] Building babylon 7.0.0
[INFO] --------------------------------[ war ]---------------------------------
[INFO] 
[INFO] >>> jetty-maven-plugin:9.4.36.v20210114:run (default-cli) > test-compile @ babylon >>>
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ babylon ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory E:\work home\mo jaja\old\tag\src\main\resources
[INFO] Copying 1 resource to E:\work home\mo jaja\old\tag\target
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:compile (default-compile) @ babylon ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 40 source files to E:\work home\mo jaja\old\tag\target\classes
[INFO] /E:/work home/mo jaja/old/tag/src/main/java/org/primefaces/babylon/component/BabylonMenuRenderer.java: E:\work home\mo jaja\old\tag\src\main\java\org\primefaces\babylon\component\BabylonMenuRenderer.java uses or overrides a deprecated API.
[INFO] /E:/work home/mo jaja/old/tag/src/main/java/org/primefaces/babylon/component/BabylonMenuRenderer.java: Recompile with -Xlint:deprecation for details.
[INFO] /E:/work home/mo jaja/old/tag/src/main/java/org/primefaces/babylon/view/ChartDemoView.java: Some input files use unchecked or unsafe operations.
[INFO] /E:/work home/mo jaja/old/tag/src/main/java/org/primefaces/babylon/view/ChartDemoView.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ babylon ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory E:\work home\mo jaja\old\tag\src\test\resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.7.0:testCompile (default-testCompile) @ babylon ---
[INFO] No sources to compile
[INFO] 
[INFO] <<< jetty-maven-plugin:9.4.36.v20210114:run (default-cli) < test-compile @ babylon <<<
[INFO] 
[INFO] 
[INFO] --- jetty-maven-plugin:9.4.36.v20210114:run (default-cli) @ babylon ---
[INFO] Logging initialized @4906ms to org.eclipse.jetty.util.log.Slf4jLog
[INFO] webAppSourceDirectory not set. Trying src\main\webapp
[INFO] Reload Mechanic: automatic
[INFO] nonBlocking:false
[INFO] Classes = E:\work home\mo jaja\old\tag\target\classes
[INFO] Configuring Jetty for project: babylon
[INFO] Context path = /babylon
[INFO] Tmp directory = E:\work home\mo jaja\old\tag\target\tmp
[INFO] Web defaults = org/eclipse/jetty/webapp/webdefault.xml
[INFO] Web overrides =  none
[INFO] web.xml file = file:///E:/work%20home/mo%20jaja/old/tag/target/web.xml
[INFO] Webapp directory = E:\work home\mo jaja\old\tag\src\main\webapp
[INFO] jetty-9.4.36.v20210114; built: 2021-01-14T16:44:28.689Z; git: 238ec6997c7806b055319a6d11f8ae7564adc0de; jvm 17.0.8.1+1
[INFO] Scanning elapsed time=516ms
[INFO] DefaultSessionIdManager workerName=node0
[INFO] No SessionScavenger set, using defaults
[INFO] node0 Scavenging every 600000ms
Dec 04, 2023 9:14:54 AM org.apache.webbeans.lifecycle.AbstractLifeCycle bootstrapApplication
INFO: OpenWebBeans Container is starting...
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/E:/work%20home/mo%20jaja/old/tag/src/main/webapp/WEB-INF/beans.xml
Dec 04, 2023 9:14:54 AM org.apache.webbeans.web.scanner.WebScannerService addWarBeansArchive
INFO: Adding information from WEB-INF/beans.xml
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/org/glassfish/jakarta.faces/2.3.14/jakarta.faces-2.3.14.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/org/apache/taglibs/taglibs-standard-impl/1.2.5/taglibs-standard-impl-1.2.5.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/org/primefaces/primefaces/13.0.0/primefaces-13.0.0.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/org/apache/xmlbeans/xmlbeans/4.0.0/xmlbeans-4.0.0.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/org/slf4j/jcl-over-slf4j/1.7.30/jcl-over-slf4j-1.7.30.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/com/zaxxer/SparseBitSet/1.2/SparseBitSet-1.2.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/E:/work%20home/mo%20jaja/old/tag/target/classes/
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/org/apache/taglibs/taglibs-standard-spec/1.2.5/taglibs-standard-spec-1.2.5.jar
Dec 04, 2023 9:14:54 AM org.apache.webbeans.corespi.scanner.AbstractMetaDataDiscovery addWebBeansXmlLocation
INFO: added beans archive URL: file:/C:/Users/PC/.m2/repository/com/github/librepdf/openpdf/1.3.24/openpdf-1.3.24.jar
Dec 04, 2023 9:14:55 AM org.apache.webbeans.config.BeansDeployer validateInjectionPoints
INFO: All injection points were validated successfully.
Dec 04, 2023 9:14:55 AM org.apache.webbeans.lifecycle.AbstractLifeCycle bootstrapApplication
INFO: OpenWebBeans Container has started, it took [689] ms.
Dec 04, 2023 9:14:55 AM com.sun.faces.config.ConfigureListener contextInitialized
INFO: Initializing Mojarra |version.string| for context '/babylon'
Dec 04, 2023 9:14:55 AM com.sun.faces.spi.InjectionProviderFactory createInstance
INFO: JSF1048: PostConstruct/PreDestroy annotations present.  ManagedBeans methods marked with these annotations will have said annotations processed.
Dec 04, 2023 9:14:56 AM com.sun.faces.config.ConfigureListener$WebConfigResourceMonitor$Monitor <init>
INFO: Monitoring file:/E:/work%20home/mo%20jaja/old/tag/src/main/webapp/WEB-INF/faces-config.xml for modifications
Dec 04, 2023 9:14:56 AM org.primefaces.webapp.PostConstructApplicationEventListener processEvent
INFO: Running on PrimeFaces 13.0.0
[INFO] Started o.e.j.m.p.JettyWebAppContext@74cad577{/babylon,[file:///E:/work%20home/mo%20jaja/old/tag/src/main/webapp/, jar:file:///C:/Users/PC/.m2/repository/org/glassfish/jakarta.faces/2.3.14/jakarta.faces-2.3.14.jar!/META-INF/resources, jar:file:///C:/Users/PC/.m2/repository/org/primefaces/primefaces/13.0.0/primefaces-13.0.0.jar!/META-INF/resources],AVAILABLE}{file:///E:/work%20home/mo%20jaja/old/tag/src/main/webapp/}
[INFO] Started ServerConnector@65d870be{HTTP/1.1, (http/1.1)}{0.0.0.0:8080}
[INFO] Started @8216ms
[INFO] Started Jetty Server
`

**Reply by Yazan-Abuawwad** [reply]
for the sample project, it works with me using jdk 1.8 
can you refer to me any project that works with Spring boot and runs the template

**Comment by Yazan-Abuawwad** [comment, marked as answer]
I solve it by using joinface    with spring boot 13  jdk 17
